test/run_test.py

```diff
@@ -1435,79 +1437,105 @@ def main():
     elif options.inductor:
         os.environ["PYTORCH_TEST_WITH_INDUCTOR"] = "1"
 
+    os.makedirs(REPO_ROOT / "test" / "test-reports", exist_ok=True)
+
     failure_messages = []
 
-    # parallel = in parallel with other files
-    # serial = this file on it's own.  The file might still be run in parallel with itself (ex test_ops)
-    selected_tests_parallel = [x for x in selected_tests if not must_serial(x.name)]
-    selected_tests_serial = [
-        x for x in selected_tests if x not in selected_tests_parallel
-    ]
-    print_to_stderr(
-        "parallel (file granularity) tests:\n {}".format(
-            "\n ".join(str(x) for x in selected_tests_parallel)
+    def run_tests(
```

Comment:
I'd normally yank this function outside to be a standalone, but doing it this way makes the diff way more readable.

I might pull it out in a future PR

Reply:
Removed